Dido's project/about.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Page</title>
+    <link
+      rel="stylesheet"
+      href="../HTML with photo background/StyleAbout.css"
```

Comment:
The file that you are trying to reach is in the same directory, so point to it as  shown

`<link rel="stylesheet" href="StyleAbout.css"/>`